tests/Microsoft.Identity.Test.Unit/AppConfigTests/ConfidentialClientApplicationBuilderTests.cs

```diff
@@ -103,7 +103,7 @@ public void CacheSynchronizationWithDefault()
                 ClientId = TestConstants.ClientId,
             };
             var app = ConfidentialClientApplicationBuilder.CreateWithApplicationOptions(options).Build();
```

Comment:
Can you add a PubliClient version here as well? i.e. create a PCA using options and make sure `CacheSynchronizationEnabled` is set to true.